dbms/src/Flash/FlashService.cpp

```diff
@@ -53,7 +53,8 @@ FlashService::FlashService(const TiFlashSecurityConfig & security_config_, Conte
           context.getGlobalContext().getSettingsRef()))
 {
     auto settings = context.getSettingsRef();
-    enable_local_tunnel = settings.enable_local_tunnel;
+    // for MPP test, we don't use local tunnel.
```

Comment:
why?
Will we support local tunnel in the future?

Reply:
Actually we have supported local tunnel in single service test. In multiple service test, we use grpc to communicate between services.

Reply:
> why? Will we support local tunnel in the future?

I will figure out how to support local tunnel across different services.